scripts/lib-franklin.js

```diff
@@ -322,14 +324,15 @@ export function updateSectionsStatus(main) {
   const sections = [...main.querySelectorAll(':scope > div.section')];
   for (let i = 0; i < sections.length; i += 1) {
     const section = sections[i];
-    const status = section.getAttribute('data-section-status');
+    const status = section.dataset.sectionStatus;
     if (status !== 'loaded') {
       const loadingBlock = section.querySelector('.block[data-block-status="initialized"], .block[data-block-status="loading"]');
       if (loadingBlock) {
-        section.setAttribute('data-section-status', 'loading');
+        section.dataset.sectionStatus = 'loading';
```

Comment:
Can a block skip straight to loading status, bypassing the `initialized` section status, which hides the block until it's loaded?

Meaning: should this section _also_ set the display to none, or is that unnecessary?

Reply:
yeah, blocks are not hidden... this is mostly due to the fact that sections really cannot be displayed before all blocks are loaded, due to the `<block>-container` on sections, which i really don't like anymore, but removing that is a different story.

for now, the unhiding works on a section by section basis, and i don't think we should change that behavior in this PR.

Reply:
scratch my previous comment... 
i didn't read things correctly...

yes, no section can go to `loading` without being `initialized` first

Reply:
Gotcha.